### DISCUSSION TITLE
Occluded objects / Oclussion / Boolean

### DISCUSSION THREAD

**Discussion by felipefierro** [google/model-viewer#4305] [Q&A]
Hi! how can I make occlusion of objects behind another object, what I need is basically a Boolean that hide whatever it's behind certain geometry (example: The floor):

Example:
[](https://felipefierro.com/hand.gif)
![hand](https://github.com/google/model-viewer/assets/134464697/effffb5d-54f2-4316-877c-ba8c87aeedb1)

**Comment by elalish** [comment]
We don't support this. I believe it's possible with three.js, so you may want to fork us to add the appropriate code. If all you want is a static mesh Boolean, consider https://github.com/elalish/manifold

**Reply by felipefierro** [reply]
Thanks you so much for your fast reply! Indeed! I need to do that with a Static mesh, but I don’t require a Boolean standard operation, but more like an object that occlude other another object and doesn’t render what’s behind, is that possible with your code?  Sorry if I’m making redundant questions but I’m an animator and I’m not very much related to code and programming. Thank you so much again!

**Reply by elalish** [reply]
Sorry, no - you'd need to write some code.
